javascript/example-customer-account--loyalty-app--ts/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "loyalty-app-react",
+  "version": "1.0.0",
+  "description": "Customer account loyalty app example",
+  "main": "index.js",
+  "scripts": {
+    "shopify": "shopify",
+    "build": "shopify app build",
+    "dev": "shopify app dev",
+    "info": "shopify app info",
+    "generate": "shopify app generate"
+  },
+  "author": "Shopify",
+  "license": "MIT",
+  "dependencies": {
+    "@shopify/app": "^3.50.0",
+    "@shopify/cli": "^3.50.0",
+    "@shopify/ui-extensions": "0.0.0-unstable-20231130134652",
```

Comment:
do we want to use `unstable` or this specific version?

Reply:
We are using this one: `0.0.0-unstable-20231024205236` the same as the react tutorials https://github.com/Shopify/customer-account-tutorials/blob/main/react/example-customer-account--wishlist--react/yarn.lock